### DISCUSSION TITLE
Action logs without fields used for action

### DISCUSSION THREAD

**Discussion by smartameer** [laravel/nova-issues#3776] [Q&A]
- Laravel Version: 8.x
- Nova Version: 3.31
- PHP Version: 8.1
- Database Driver & Version: Mysql
- Operating System and Version: *
- Browser type and version: *

### Description:
Is there any way to store the Action event only without fields being stored?
With current implementation its logging submitted form fields

**Comment by crynobone** [comment]
Unable to reproduce the issue

![image](https://user-images.githubusercontent.com/172966/158176092-ce02dda2-00c0-480a-8a2f-ca714ba0b6c1.png)

If you are reporting a bug, please provide follow the bug report template and outline the description of how to reproduce your issue, best with reproducing code.

**Reply by smartameer** [reply]
Hello @crynobone,
My apologies that i didn't write procedure to reproduce

I am using the action with fields and fields are being stored part of logs

<img width="763" alt="image" src="https://user-images.githubusercontent.com/5998873/158185381-47ccb646-6fec-4a20-bfae-5e2202589c5f.png">

![image](https://user-images.githubusercontent.com/5998873/158185760-c8bdcc05-95a5-4412-b174-c654aa7a797c.png)

**Reply by crynobone** [reply]
You should disable Action Events on such Action: <https://nova.laravel.com/docs/3.0/actions/defining-actions.html#disabling-the-action-log>

**Reply by smartameer** [reply]
Ok. There is no way to disable the fields being pushed to db. Event is always with fields if available.
Thank you @crynobone
